openverse_catalog/dags/providers/provider_api_scripts/museum_victoria.py

```diff
@@ -145,24 +146,22 @@ def _get_media_info(media_data):
 
 
 def _get_image_data(media):
-    image_url = None
-    height, width = None, None
+    media_data = {}
+    filetype = None
     if "large" in media.keys():
-        image_url = media.get("large").get("uri")
-        height = media.get("large").get("height")
-        width = media.get("large").get("width")
-
+        media_data = media.get("large")
     elif "medium" in media.keys():
-        image_url = media.get("medium").get("uri")
-        height = media.get("medium").get("height")
-        width = media.get("medium").get("width")
-
+        media_data = media.get("medium")
     elif "small" in media.keys():
-        image_url = media.get("small").get("uri")
-        height = media.get("small").get("height")
-        width = media.get("small").get("width")
-
-    return image_url, height, width
+        media_data = media.get("small")
+
+    image_url = media_data.get("uri")
+    height = media_data.get("height")
+    width = media_data.get("width")
+    filesize = media_data.get("size")
+    if image_url:
+        filetype = extract_filetype(image_url, "image")
```

Comment:
Hmm, this is making me wonder if `extract_filetype` should be able to accept `None` for `image_url`, so we wouldn't have to do an explicit `if image_url` check every time before calling it in our provider scripts 😅 What do you think?

Reply:
Actually, since the `image_url` is a required field, we should short-circuit further actions if `image_url` is `None`. Great catch!

Reply:
It wasn't as easy to short-circuit this, because `image_url` is in a nested dictionary. But I did add indentation to the next couple of lines so that we only try to get the `width`, `height` and `filesize` if the `image_url` is not `None`. 
Oh, and I guess I did not update this script to the changes from the `helper` PR, because `extract_filetype` (from the media URL) should be called by the `ImageStore` anyways. I've updated it now, sorry!